open_schools_platform/organization_management/circles/views.py

```diff
@@ -63,6 +64,11 @@ class GetCirclesApi(ApiAuthMixin, ListAPIView):
         tags=[SwaggerTags.ORGANIZATION_MANAGEMENT_CIRCLES],
     )
     def get(self, request, *args, **kwargs):
+        data = request.GET.dict()
+        if 'student' in data.keys():
+            get_student(filters={"id": data['student']}, user=request.user)
```

Comment:
should we check permission there?
we don't use student_profile information outside this organization

Reply:
I put this check here because it would be strange if request.user, who doesn't have anything to do with provided student would be able to see list of his circles.

I mean, if request.user don't have same family with student's student_profile, or he is not his owner or he is not an employee of student's circle organization. In this case, without this check, he will be able to get list of student's circles (at least for particular organization).

What do you think?